src/components/SelectionList/BaseSelectionList.js

```diff
@@ -237,34 +286,65 @@ function BaseSelectionListRadio(props) {
+                            >
+                                <Checkbox
+                                    accessibilityLabel={translate('workspace.people.selectAll')}
+                                    isChecked={flattenedSections.allSelected}
+                                    onPress={props.onSelectAll}
+                                />
+                                <View style={[styles.flex1]}>
+                                    <Text style={[styles.textStrong, styles.ph5]}>{translate('workspace.people.selectAll')}</Text>
+                                </View>
+                            </PressableWithFeedback>
+                        )}
                         <SectionList
                             ref={listRef}
                             sections={props.sections}
+                            renderSectionHeader={renderSectionHeader}
                             renderItem={renderItem}
                             getItemLayout={getItemLayout}
                             onScroll={props.onScroll}
                             onScrollBeginDrag={props.onScrollBeginDrag}
                             keyExtractor={(item) => item.keyForList}
+                            onLayout={() => scrollToIndex(focusedIndex, false)}
                             extraData={focusedIndex}
                             indicatorStyle="white"
                             keyboardShouldPersistTaps="always"
                             showsVerticalScrollIndicator={false}
-                            OP
                             initialNumToRender={12}
                             maxToRenderPerBatch={5}
                             windowSize={5}
                             viewabilityConfig={{viewAreaCoveragePercentThreshold: 95}}
-                            onLayout={() => scrollToIndex(focusedIndex, false)}
+                            testID="selection-list"
                         />
+                        {shouldShowConfirmButton && (
+                            <FixedFooter>
+                                <Button
+                                    success
+                                    style={[styles.w100]}
+                                    text={confirmButtonText}
+                                    onPress={props.onConfirm}
+                                    pressOnEnter
+                                    enterKeyEventListenerPriority={1}
+                                />
+                            </FixedFooter>
+                        )}
                     </View>
                 )}
             </SafeAreaConsumer>
         </ArrowKeyFocusManager>
     );
 }
 
-BaseSelectionListRadio.displayName = 'BaseSelectionListRadio';
-BaseSelectionListRadio.propTypes = propTypes;
-BaseSelectionListRadio.defaultProps = selectionListRadioDefaultProps;
+BaseSelectionList.displayName = 'BaseSelectionList';
+BaseSelectionList.propTypes = propTypes;
+BaseSelectionList.defaultProps = selectionListDefaultProps;
```

Comment:
If we use prop destructuring with default values we don't need this anymore -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring